src/ServiceManagement/Compute/ComputeManagement/Properties/AssemblyInfo.cs

```diff
@@ -19,8 +19,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Compute Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Virtual Machines and Hosted Services.")]
 
-[assembly: AssemblyVersion("13.0.0.0")]
-[assembly: AssemblyFileVersion("13.0.0.0")]
+[assembly: AssemblyVersion("14.0.0.0")]
```

Comment:
Are there breakign changes in this update?  If the change is only new API surface, then the new AssemblyFileVersion should be 13.1.0.0 and the AssemblyVersion should be unchanged.